trpj00-sp2

JanTrpkos opened the pull request:
https://eso.vse.cz/~trpj00/sp2/

JanTrpkos committed:
trpj00-sp2

https://eso.vse.cz/~trpj00/sp2/

JanTrpkos committed:
version 2

Changed...

nvbach91 reviewed (changes requested):
- elementy z HTML nevybirat opakovane v prubehu programu, ale vybirat pouze jednou a ukladat je do promennych
- funkce deklarovat jako lokalni pomoci `const`
- v async/await vetsinou byva try/catch blok
- nezasahovat do DOM v cyklu (pridavani novych elementu, zmena atributu, obsahu, atd.)
- soubory davat do vhodne struktury slozek
- lepe nastylovat layout
- pridat moznost prepinani mezi Celsius a Fahrenheit a Kelvin
- aplikace by mela si zapamatovat posledni nalezene mesto nebo nejaky defaultni mesto v pripade ze geolokace neni k dispozici
-

JanTrpkos committed:
version 03

elementy z HTML nevybirat opakovane v prubehu programu, ale vybirat pouze jednou a ukladat je do promennych (opraveno)
funkce deklarovat jako lokalni pomoci const (opraveno)
v async/await vetsinou byva try/catch blok (opraveno)
nezasahovat do DOM v cyklu (pridavani novych elementu, zmena atributu, obsahu, atd.) (opraveno)
soubory davat do vhodne struktury slozek (opraveno)
lepe nastylovat layout (opraveno)
pridat moznost prepinani mezi Celsius a Fahrenheit a Kelvin (přidáno)
aplikace by mela si zapamatovat posledni nalezene mesto nebo nejaky defaultni mesto v pripade ze geolokace neni k dispozici (Aplikace reflektuje pokud uživatel neposkytne polohu)

JanTrpkos committed:
Delete www/trpj00/sp2/style.css

JanTrpkos committed:
Delete www/trpj00/sp2/JS/main.js

nvbach91 commented:
vratte prosim aktualni kod ze dneska do Pull Requestu

JanTrpkos committed:
version 3.1

fixing missing file

JanTrpkos committed:
Delete www/trpj00/sp2/main.js
